Backend/Interview.Backend/RoomReviews/RoomReviewController.cs

```diff
@@ -80,7 +81,25 @@ public async Task<ActionResult<RoomReviewDetail>> Create([FromBody] RoomReviewCr
     }
 
     /// <summary>
-    /// Update a review by id.
+    /// Upsert a review for a room
+    /// </summary>
+    /// <param name="request">User Request.</param>
+    /// <param name="currentUserAccessor">Current user accessor</param>
+    /// <returns>Review details</returns>
+    [Authorize]
+    [HttpPut("/upsert")]
```

Comment:
It definitely needs a slash at the beginning of the name?

Reply:
No, but it is correct.